kalite/control_panel/templates/control_panel/data_export.html

```diff
@@ -3,7 +3,7 @@
 {% load staticfiles %}
 {% load include_block %}
 
-{% block facility_active %}facility-active active-tab active-nav{% endblock facility_active %}
+{% block facility_active %}active{% endblock facility_active %}
```

Comment:
Can you add `{% block i18n_do_not_translate %}` and `{% endblock i18n_do_not_translate %}` tags surrounding this section?